src/main/java/com/rackspace/ceres/app/services/DownsampleProcessor.java

```diff
@@ -137,19 +138,22 @@ private void processPartitions(int job) {
     scheduled = partitionsToProcess(job)
         .mapToObj(partition -> taskScheduler.schedule(
             () -> processPartition(partition),
-            Instant.now().plus(randomizeDelay())
+            Instant.now().plusSeconds(deltaSeconds(job, partition))
         )).collect(Collectors.toList());
   }
 
+  private long deltaSeconds(int job, int partition) {
+    // We want to space out all the partitions evenly in time between jobs
+    long processPeriodSeconds = downsampleProperties.getDownsampleProcessPeriod().toSeconds();
+    long deltaSeconds = processPeriodSeconds/16;
```

Comment:
Is 16 here the number of jobs?

Reply:
yes, it's the number of partitions to be spaced out evenly.